ModelBundle/Validator/Constraints/CheckMainAliasPresenceValidator.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace OpenOrchestra\ModelBundle\Validator\Constraints;
+
+use OpenOrchestra\ModelInterface\Model\SiteInterface;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+class CheckMainAliasPresenceValidator extends ConstraintValidator
+{
+    /**
+     * Checks if the passed value is valid.
+     *
+     * @param SiteInterface $value The value that should be validated
+     * @param Constraint    $constraint The constraint for the validation
+     */
+    public function validate($value, Constraint $constraint)
+    {
+        if (!$value->getMainAlias()->isMain()) {
```

Comment:
Are you sure that line 276 is an error? In this case, we should not return null but an exception as main alias is mandatory. What about the BC Break for the integrators and theirs fixtures? @itkg-ngilain